app/controllers/api/v1/rents_controller.rb

```diff
@@ -30,6 +31,11 @@ def search_param
         return { user_id: params['user_id'] } if params['user_id'].present?
         return { book_id: params['book_id'] } if params['book_id'].present?
       end
+
+      # Summary: Send email on creation service
+      def send_mail_on_creation(rent)
+        RentMailer.success_rent_email(current_user, rent, rent.book).deliver_later
```

Comment:
When you send something to sidekiq to perform later, you need to send as little info as posible, because what you're actually doing is using redis to store the data so that later sidekiq fetches it and performs the task.

Here, for example, we are serializing the entire user, book and rent entity, which uses a lot of memory. Imagine this info being saved for every Rent created in a big website, it will soon turn into storing GBs of data which could be avoided. 
Generally, it is better to just send the rent id and then fetch whatever extra info you need in the mailer.